src/gluonts/dataset/artificial/generate_synthetic.py

```diff
@@ -24,6 +24,7 @@
     ComplexSeasonalTimeSeries,
     ConstantDataset,
 )
+from gluonts.dataset.common import ArrowWriter
```

Comment:
Can we do this in a second PR?

Reply:
I'd prefer not to invest time in splitting the PR if not necessary.